src/components/form/index.tsx

```diff
@@ -106,7 +128,10 @@ export class SmoothlyForm implements Clearable, Submittable, Editable {
 										: {
 												method,
 												url: action,
-												...(this.value && { header: { contentType: "application/json" }, body: this.value }),
+												...(this.value && {
+													header: { contentType: this.binary ? "multipart/form-data" : "application/json" },
```

Comment:
shouldn't cloudly http deduce if you're sending form data? then we don't need the binary thing

Reply:
Ideally yes, but since we are sending a file that is nested somewhere under our object cloudly http is not smart enough to see that. It instead sees an object and thinks its JSON data. So we have to specify it.